sklearn/linear_model/tests/test_base.py

```diff
@@ -607,31 +608,32 @@ def test_sparse_preprocess_data_offsets(global_random_seed):
     assert_array_almost_equal(yt, y - np.mean(y, axis=0))
 
 
-def test_csr_preprocess_data():
+@pytest.mark.parametrize("csr_container", CSR_CONTAINERS)
+def test_csr_preprocess_data(csr_container):
     # Test output format of _preprocess_data, when input is csr
     X, y = make_regression()
     X[X < 2.5] = 0.0
-    csr = sparse.csr_matrix(X)
+    csr = csr_container(X)
     csr_, y, _, _, _ = _preprocess_data(csr, y, True)
     assert csr_.getformat() == "csr"
```

Comment:
I think this should resolve the CI error.
```suggestion
    assert csr_.format == "csr"
```

Reply:
Oh didn't notice the CI is complaining. Thanks for the reminder.